src/components/WorkspaceControlPanel.js

```diff
@@ -0,0 +1,77 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import Display from './Display';
+import ConnectedManifestForm from './ManifestForm';
+import ConnectedManifestListItem from './ManifestListItem';
+import ns from '../config/css-ns';
+
+/**
+ * Provides the panel responsible for controlling the entire workspace
+ */
+class WorkspaceControlPanel extends Component {
+  /**
+   * constructor -
+   */
+  constructor(props) {
+    super(props);
+    this.state = {
+      lastRequested: '',
+    };
+
+    this.setLastRequested = this.setLastRequested.bind(this);
+  }
+
+  /**
+   * setLastRequested - Sets the state lastRequested
```

Comment:
I think that docstring could be a bit more descriptive, what exactly is `lastRequested`?

Reply:
Originally this was a flag to allow us to display only the information for the most recently requested manifest. We can probably just delete this at this point, as the side panel designs don't really require this behaviour.

Reply:
I agree @jbaiter.  I was simply moving this comment w/ its accompanying code from another place to here in my component extraction, so not really sure I'm the best person to describe what the intent of the original author was.